packages/app/cypress/e2e/runner/retries.mochaEvents.cy.ts

```diff
@@ -2,7 +2,7 @@ import { runSpec } from './support/spec-loader'
 import { runCypressInCypressMochaEventsTest } from './support/mochaEventsUtils'
 import { snapshots } from './retries.mochaEvents.snapshots'
 
-describe('src/cypress/runner retries mochaEvents', { retries: 0 }, () => {
+describe('src/cypress/runner retries mochaEvents', { retries: 0, defaultCommandTimeout: 7500 }, () => {
```

Comment:
@lmiller1990 Can we add comments for all these timeout increases, that they are for windows tests so people in the future will understand why this is increased. Ideally I’d like to understand why these are even necessary though.

Reply:
Ok - we can probably make these conditional potentially, and add a comment, until we find out what (if there is an actual reason) makes these slow in windows. My feeling is general windows slowness, and since these use cypress-in-cypress, which is a lot more resource intensive than the original `runIsolatedCypress`, the compounding is just slower on windows (everything is, even the unit tests in 9.x are almost 2x slower in windows).

I'll log an issue for this to be looked into.

Reply:
https://github.com/cypress-io/cypress/pull/21055/commits/167a43627b5989c19dd635df851940c34b82d0e6

Also made issue to track in Jira, I sent it in our internal chat.